src/components/Sidebar/PostDetails/NewCommentForm.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+import { addComment } from '../../../utils/fetchFunctions';
+import {
+  ActivePostContext,
+  CommentListContext,
+  ErrorsContext,
+} from '../../../utils/Store';
+import { ErrorTypes } from '../../../types/ErrorTypes';
+
+enum Keys {
+  nameField = 'nameField',
+  emailField = 'emailField',
+  textArea = 'textArea',
+}
+
+export const NewCommentForm: React.FC = () => {
+  const [isErrorField, setIsErrorField] = useState({
+    nameField: false,
+    emailField: false,
+    textArea: false,
+  });
+
+  const [inputQueries, setInputQueries] = useState({
+    nameField: '',
+    emailField: '',
+    textArea: '',
+  });
+
+  const [isProcessing, setIsProcessing] = useState(false);
```

Comment:
Consider renaming 'isProcessing' to 'isLoading' or 'isLoaded' to follow the checklist requirement: 'don't use isLoad, it can be isLoading or isLoaded'.